CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# suitcase-csv changes
+
+## v0.2.1 (2019-08-16)
+
+## v0.2.0 (2019-05-28)
+
+## v0.1.1 (2019-03-07)
+
+## v0.1.0 (2019-03-07)
+
+## v0.1.0b1 (2019-02-08)
```

Comment:
Do we think it's enough to post the versions and dates? I think there should be some summaries of the changes. You can assemble them from the titles of the PRs for each release. Something like https://github.com/conda/conda-build/blob/master/CHANGELOG.txt would be nice to adopt.

On the other hand, I didn't find the changelogs for numpy, scipy and pandas. Is it still a mainstream to have the changelogs?

Reply:
>  I think there should be some summaries of the changes.

It would be reasonable to ask people who have been more involved in suitcase development to write these, as it's harder for @ke-zhang-rd to absorb and succinctly summarize the intent behind all the changes. That said, I would certainly not complain if Ke added some of these.

> On the other hand, I didn't find the changelogs for numpy, scipy and pandas. Is it still a mainstream to have the changelogs?

Yes.

https://docs.scipy.org/doc/numpy/release.html
https://docs.scipy.org/doc/scipy/reference/release.html
https://pandas.pydata.org/docs/whatsnew/index.html

Additionally, the prereleases like `v0.1.0b1` should not have sections here.

Reply:
OK, I see it's in the built docs, but there are no `CHANGELOG.*` files. I guess having only one source of the truth is reasonable here.

Reply:
>Additionally, the prereleases like v0.1.0b1 should not have sections here.

Should I also remove `rc`, `a` version?

Reply:
Yes, those are also prereleases.

Reply:
removed.